FAST_LIO/src/laserMapping.cpp

```diff
@@ -1322,7 +1322,7 @@ int main(int argc, char** argv)
             msg_body_pose.pose.orientation.w = geoQuat.w;
 
             pose_publisher.publish(msg_body_pose);
-            mavros_pose_publisher.publish(msg_body_pose);
+            //mavros_pose_publisher.publish(msg_body_pose);
```

Comment:
Let's delete it, we won't be using this publisher. And don't forget to delete any other refs to mavros_pose_publisher